scripts/packages/fedora/bazel.spec

```diff
@@ -0,0 +1,43 @@
+Name: bazel
+Version: devel
+Release: 1
+Summary: Correct, reproducible, and fast builds for everyone.
+URL: https://bazel.build
+License: Apache License, v2.0
+
+Source0: bazel
+Source1: bazel-real
+Source2: bazel.bazelrc
+Source3: bazel-complete.bash
+
+Requires: java-1.8.0-openjdk-headless
+
+%description
+Bazel is a build tool that builds code quickly and reliably. It is used to build the majority of Google's software, and thus it has been designed to handle build problems present in Google's development environment.
+
+%global _enable_debug_package 0
+%global debug_package %{nil}
+%global __os_install_post /usr/lib/rpm/brp-compress %{nil}
+
+%prep
+
+%build
+
+%install
+export DONT_STRIP=1
+mkdir -p %{buildroot}%{_bindir}/
+install -m 755 bazel %{buildroot}%{_bindir}/bazel
+install -m 755 bazel-real %{buildroot}%{_bindir}/bazel-real
+mkdir -p %{buildroot}%{_sysconfdir}/
+install -m 644 bazel.bazelrc %{buildroot}%{_sysconfdir}/bazel.bazelrc
+mkdir -p %{buildroot}%{_sysconfdir}/bash_completion.d
+install -m 644 bazel-complete.bash %{buildroot}%{_sysconfdir}/bash_completion.d/bazel
+
+%files
+%{_bindir}/bazel
+%{_bindir}/bazel-real
+%{_sysconfdir}/bazel.bazelrc
+%{_sysconfdir}/bash_completion.d/bazel
+
+%changelog
+# TODO: Include changelog.
```

Comment:
Right now, the changelog is empty and the version is "devel".

Here are my next steps on this:
1. Submit this PR. :)
2. Write a pkg_rpm that generates the spec file, possibly from a template, instead of using a static one.
3. Have two targets: bazel-devel-rpm and bazel-release-rpm. The release target will include the changelog and version from parsing the same sources of data as the debian package targets.
4. Get a RHEL worker in GCE and start building RPMs during releases.